modules/utxo-lib/src/networks.ts

```diff
@@ -116,6 +117,15 @@ export const networks: Record<NetworkName, Network> = {
     wif: 0xef,
     coin: coins.BTC,
   },
+  bitcoinPublicSignet: {
```

Comment:
Source for constants: https://github.com/bitcoin/bitcoin/blob/2c79abc7ad4850e9e3ba32a04c530155cda7f980/src/kernel/chainparams.cpp#L387

Reply:
pls add this link into jsdoc https://github.com/bitcoin/bitcoin/blob/v27.1/src/kernel/chainparams.cpp#L287-L290